README.md

```diff
@@ -74,6 +74,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Mattermost](https://github.com/mattermost/mattermost-server/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22Up+For+Grabs%22+label%3A%22Difficulty%2F1%3AEasy%22+label%3A%22Tech%2FGo%22+) _(label: Up For Grabs, Difficulty/1:Easy, Tech/Go)_ <br> Open source Slack-alternative in Golang and React
 - [Docker/CLI](https://github.com/docker/cli/labels/exp%2Fbeginner) _(label: exp/beginner)_ <br> The Docker CLI
 - [Kubernetes](https://github.com/kubernetes/kubernetes/labels/good%20first%20issue) _(label: good first issue)_ <br> Production-Grade Container Scheduling and Management System
+- [TiDB](https://github.com/pingcap/tidb/labels/for%20new%20contributors) _(label: for new contributors, Difficulty: Introductory)_ <br> A distributed scalable Hybrid Transactional and Analytical Processing (HTAP) database
```

Comment:
Doesn't look like correct link

Reply:
Thanks! @vvscode ! Could you please elaborate on what's wrong with this link? It seems working from my side. I changed the description to make it consistent with the others.